projects/core/src/event/event.service.ts

```diff
@@ -39,11 +32,7 @@ interface EventMeta<T> {
   providedIn: 'root',
 })
 export class EventService {
-  constructor(
-    // TODO: #10896 - remove this
-    /** @deprecated @since 3.1 - this will be removed in 4.0 */ @Optional()
-    protected featureConfigService?: FeatureConfigService
-  ) {}
+  constructor() {}
```

Comment:
[nitpick] do we leave the empty constructor?

Reply:
Good point, removed in order to save a couple of bytes ;)